frontend/components/instructor/SessionDetailContent.tsx

```diff
@@ -154,7 +154,13 @@ export default function SessionDetailContent({
           isMobile ? "gap-[24px]" : "gap-[16px]"
         } `}
       >
-        <InfoCard className="stat-info-card">
+        <InfoCard
+          className={`stat-info-card ${
+            isMobile
+              ? "h-[160px] border border-[1px] border-surface_border_tertiary px-[16px] py-[24px]"
```

Comment:
delete "border" here i don't think it does anything
also can you double check that you need fixed height for mobile?